### PR TITLE
SPLAT-868 - plugin: add artifacts collector instance

### PR DESCRIPTION
**Pull request by mtulio**
Introduce to plugin artifacts collector, to collect:
- e2e suite list for openshift and kubernetes conformance
- must-gather

https://issues.redhat.com/browse/SPLAT-868

Relatest to:

- https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/32

Blocked by:

- [x] https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pull/27

### CODE REVIEW

**Comment by mtulio** [comment]
@bostrt ptal?

File: openshift-tests-provider-cert/plugin/executor.sh [hunk 1]
```diff
@@ -62,6 +62,21 @@ if [[ -n "${CERT_TEST_SUITE}" ]]; then
     os_log_info "openshift-tests finished[$?]"
     set +x
 
+# Collect artifacts post-execution
+elif [[ "${CERT_LEVEL}" == "99" ]]; then
```
Comment (mtulio) [inline openshift-tests-provider-cert/plugin/executor.sh:66]: Depends on https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pull/21/files

**Comment by mtulio** [comment]
@bostrt rebased and ready for review! ptal? =)

File: openshift-tests-provider-cert/plugin/global_fn.sh [hunk 2]
```diff
@@ -54,6 +54,11 @@ init_config() {
         CERT_TEST_SUITE="openshift/conformance"
         PLUGIN_BLOCKED_BY+=("10-openshift-kube-conformance")
 
+    elif [[ "${CERT_LEVEL:-}" == "${PLUGIN_ID_OPENSHIFT_ARTIFACTS_COLLECTOR}" ]]
+    then
+        PLUGIN_NAME="99-openshift-conformance-validated"
```
Comment (bostrt) [inline openshift-tests-provider-cert/plugin/global_fn.sh:59]: I think this needs to be renamed

Comment (mtulio) [reply openshift-tests-provider-cert/plugin/global_fn.sh:59]: the `CERT_LEVEL` is also incorrect for the recent changes on the `main`. Will fix all references

**Comment by bostrt** [comment]
after testing (see https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/32#issuecomment-1340064713) this looks good to me!

/lgtm

**Review by bostrt — APPROVED** [review]

**Comment by mtulio** [comment]
@bostrt rebased. Please lgtm it again?

**Pull request merged**